scripts/kbdcfg_to_kbdxml.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3
+
+"""
+README
+
+1. Open the terminal and cd to the directory containing the *kbd.cfg files
+   that you want to translate.
```

Comment:
I am afraid most of our users are not able to to do this. With this approach we will likely receive bug reports like:
"Custom Keyboard mapping is broken after update"
We should move this migration code to the C++ domain and call It automatically for custom mappings. 
It would be bad if a user updates Mixxx and notices the missing custom keys during a gig.